pkg/apiserver/handlers.go

```diff
@@ -32,6 +32,12 @@ import (
 	"github.com/golang/glog"
 )
 
+var apiPrefixes = util.NewStringSet("api")
+
+func AddAPIPrefix(prefixes ...string) {
+	apiPrefixes.Insert(prefixes...)
```

Comment:
No global statics.  We need to fix this at the time we create handlers, not via static code.